packages/cw721/src/msg.rs

```diff
@@ -140,21 +192,60 @@ pub enum Cw721QueryMsg<TMetadataExtension> {
     #[returns(NumTokensResponse)]
     NumTokens {},
 
-    #[returns(CollectionInfo)]
+    #[deprecated(
+        since = "0.19.0",
+        note = "Please use GetCollectionInfoAndExtension instead"
+    )]
+    #[returns(CollectionInfoAndExtensionResponse<TCollectionExtension>)]
+    /// Deprecated: use GetCollectionInfoAndExtension instead! Will be removed in next release!
     ContractInfo {},
 
+    /// Returns `CollectionInfoAndExtensionResponse`
+    #[returns(CollectionInfoAndExtensionResponse<TCollectionExtension>)]
+    GetCollectionInfoAndExtension {},
+
+    /// returns `AllInfoResponse` which contains contract, collection and nft details
+    #[returns(AllInfoResponse)]
+    GetAllInfo {},
+
+    /// Returns `CollectionExtensionAttributes`
+    #[returns(CollectionExtensionAttributes)]
+    GetCollectionExtensionAttributes {},
+
+    #[deprecated(since = "0.19.0", note = "Please use GetMinterOwnership instead")]
     #[returns(Ownership<Addr>)]
+    /// Deprecated: use GetMinterOwnership instead! Will be removed in next release!
     Ownership {},
 
+    /// Return the minter
+    #[deprecated(since = "0.19.0", note = "Please use GetMinterOwnership instead")]
+    #[returns(MinterResponse)]
+    /// Deprecated: use GetMinterOwnership instead! Will be removed in next release!
+    Minter {},
+
+    #[returns(Ownership<Addr>)]
+    GetMinterOwnership {},
+
+    #[returns(Ownership<Addr>)]
+    GetCreatorOwnership {},
+
     /// With MetaData Extension.
     /// Returns metadata about one particular token, based on *ERC721 Metadata JSON Schema*
     /// but directly from the contract
-    #[returns(NftInfoResponse<TMetadataExtension>)]
+    #[returns(NftInfoResponse<TNftExtension>)]
     NftInfo { token_id: String },
+
+    #[returns(Option<NftInfoResponse<TNftExtension>>)]
+    GetNftByExtension {
```

Comment:
None of the other getters use the `Get` prefix and its typically not used in Rust. Any reason for using it here?

Reply:
In cw72 v18 there there is an `extension` message for both: execute and query.
So ts-cogegen generates compile errors due to name clash. So I started with new messages prefixing with `Get`:
`GetCollectionInfoAndExtension`, `GetAllInfo`, `GetMinterOwnership`, `GetCreatorOwnership`, `GetNftByExtension`, `GetCollectionExtension`, and `GetWithdrawAddress`.

Only `Extension` did not get renamed - for backward compat reasons - here I renamed `UpdateExtension` execute msgs, since default operation is a no-op.

Please also check #169